sdk/cosmos/ci.yml

```diff
@@ -45,3 +45,8 @@ stages:
         - name: azure-cosmos
           groupId: com.azure
           safeName: azurecosmos
+      AdditionalModules:
```

Comment:
@kushagraThapar @mitchdenny @weshaggard 
I realize that we're doing this to allow Cosmos pipelines to pass but the examples and benchmark libraries are supposed to be going away which means that these specific edits should ultimately be going away.